elasticdl/python/common/model_handler.py

```diff
@@ -61,57 +66,93 @@ def get_model_to_train(self, model):
         an elasticdl.layers.Embedding layer in ParameterServerStrategy.
         """
         if type(model) == tf.keras.Sequential or model._is_graph_network:
-            model = self._clone_model_for_sequential_and_functional(model)
+            model = self._replace_embedding_layer_to_clone_model(
+                model, tf.keras.layers.Embedding, Embedding
+            )
         else:
-            model = self._replace_embedding_attribute_for_subclass(model)
+            model = self._replace_embedding_attribute_for_subclass(
+                model, tf.keras.layers.Embedding, Embedding
+            )
         return model
 
-    def get_model_to_export(self, model, dataset):
+    def get_model_to_export(self, model, trained_params, dataset):
         """
-        To export model for tf-serving by tf.saved_model.save:
-        1. Add inputs and outputs to the model.
-        2. Restore Keras model and replace embedding parameters with trained
-        model.
+        Get the model which can be exported a SavedModel by
+        tf.saved_model.save.
```

Comment:
Put tilts around code (same in other places)

Reply:
Use `tf.saved_model.save`  in docstring?